main.py

```diff
@@ -84,6 +86,35 @@ def __init__(self, sqs_apps, elb_apps, scheduled_job_apps):
         self.last_scale_up = {}
         self.last_scale_down = {}
 
+    def load_schedule(self):
+        try:
+            schedule = None
+            with open("schedule.yml") as f:
```

Comment:
Would a more defensive stance be better?

if os.path.exisits("schedule.yml"):
 open